app/components/link_disabled_by_tooltip_component.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+class LinkDisabledByTooltipComponent < ApplicationComponent
+  def initialize(enabled:, text:, path:, tooltip:, method: :get, class_list: nil)
+    @enabled = !!enabled
```

Comment:
Does this equate to `true` by default?

Reply:
This is a Ruby idiom to ensure you've got a boolean value. It's a required keyword, so it doesn't default to anything